.github/workflows/build.yaml

```diff
@@ -0,0 +1,478 @@
+    runs-on: ubuntu-latest
+    needs:
+      - pre-build-updater
+    container:
+      image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.61
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - uses: ./.github/actions/job-preamble
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: updater-build
+
+      - name: Unpack updater build
+        run: |
+          tar xvzf updater-build.tgz
+
+      - name: genesis-dump
+        run: |
+          source ./scripts/ci/lib.sh
+          generate_genesis_dump
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: genesis-dump
+          path: /tmp/genesis-dump/genesis-dump.zip
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: vuln-dump
+          path: /tmp/vuln-dump
+
+  generate-db-dump:
+    runs-on: ubuntu-latest
+    needs:
+      - generate-genesis-dump
+    container:
+      image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.61
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - uses: ./.github/actions/job-preamble
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: updater-build
+
+      - name: Unpack updater build
+        run: |
+          tar xvzf updater-build.tgz
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: genesis-dump
+          path: /tmp/genesis-dump
+
+      - name: db-dump
+        run: |
+          source ./scripts/ci/lib.sh
+          generate_db_dump
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: db-dump
+          path: /tmp/postgres/pg-definitions.sql.gz
+
+  generate-scanner-bundle:
+    runs-on: ubuntu-latest
+    needs:
+      - generate-genesis-dump
+      - pre-build-scanner
+    container:
+      image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.61
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - uses: ./.github/actions/job-preamble
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: scanner-build
+
+      - name: Unpack scanner build
+        run: |
+          tar xvzf scanner-build.tgz
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: vuln-dump
+          path: /tmp/vuln-dump
+
+      - name: Generate OSS notice
+        run: make ossls-notice
+
+      - name: Get genesis dump
+        run: |
+          source ./scripts/ci/lib.sh
+          get_genesis_dump
+
+      - name: Make bundle
+        run: image/scanner/rhel/create-bundle.sh image/scanner image/scanner/rhel
+
+      - name: Bundle the bundle to preserve permissions # TODO(DO NOT MERGE): word this better?
```

Comment:
lol it's accurate. it's honestly ok with me. you can also switch the first bundle with archive